lib/brains/forms.py

```diff
@@ -1,5 +1,40 @@
 from django import forms
+from django.core.exceptions import ObjectDoesNotExist
+
+from lib.buyers.models import Buyer
 
 
 class BuyerForm(forms.Form):
     uuid = forms.CharField(max_length=255)
+
+
+class PaymentMethodForm(forms.Form):
+    buyer_uuid = forms.CharField(max_length=255)
+    nonce = forms.CharField(max_length=255)
+
+    def clean_buyer_uuid(self):
+        data = self.cleaned_data
+
+        try:
+            self.buyer = Buyer.objects.get(uuid=data['buyer_uuid'])
+        except ObjectDoesNotExist:
+            raise forms.ValidationError('Buyer does not exist.')
```

Comment:
It really depends where those values are going to be shown. I've looked at the ones in solitude and I don't think a single one of those will ever go up to an end user. I think they occurred during development and things were confusing and it became it stop gap.

The problem is I think that approach has to be applied to everything or nothing.

Perhaps there's other ways of doing this and it really depends where the logic is going to occur. It feels like its going to be in payments-service again and what happens there will be interesting.